tests/robustness/validate/watch.go

```diff
@@ -56,19 +57,19 @@ func validateWatch(lg *zap.Logger, cfg Config, reports []report.ClientReport, ev
 		if err != nil {
 			return err
 		}
+		err = validateBookmarkable(lg, r)
+		if err != nil {
+			return err
+		}
 		if eventHistory != nil {
```

Comment:
although it is trivial, can you add a test case where eventHistory is empty but there are multiple progress notify events?

Reply:
A passing case or a failing one?

Reply:
a passing case. And the test case for 2 progressNotify responses with the wrong rev order is also missing.

Reply:
Good point!